modules/virt-virtctl-commands.adoc

```diff
@@ -228,20 +228,20 @@ This command removes the association between the VM and the PVC, so that the mem
 
 You use `virtctl` to add or remove resources from running VMs and VMIs.
 
-.Hot plug and hot-unplug commands
+.Hot plug and hot unplug commands
 [width="100%",cols="1a,2a",options="header"]
 |===
 |Command |Description
 |`virtctl addvolume <vm_name> --volume-name=<datavolume_or_PVC> [--persist] [--serial=<label>]`
-|Hot plug a virtual disk by attaching a data volume or persistent volume claim (PVC) to a running VM.
+|Hot plug a data volume by attaching a data volume or persistent volume claim (PVC) to a running VM.
```

Comment:
```suggestion
|Hot plug a data volume or persistent volume claim (PVC) to a running VM.
```

Reply:
@apinnick I'm treating the term "hot plug" as redundant to the term "attaching".

Reply:
LGTM